### DISCUSSION TITLE
Is exists VS pluggin for show ETW  events in IDE like PerfView?

### DISCUSSION THREAD

**Discussion by P9avel** [dotnet/runtime#94257] [General]
Hi. I am want to debug app step by step and  show ETW events in nesteed  window in IDE. Possible exists?

**Comment by gfoidl** [comment]
Is [Events viewer](https://learn.microsoft.com/en-us/visualstudio/profiling/events-viewer?view=vs-2022) what you are looking for?

**Comment by P9avel** [comment]
Event viewer show eveents only after stop. I am need debug app step-by-step and see events
